docs/azure/includes/dotnet-all.md

```diff
@@ -1,450 +1,453 @@
+| Azure.Core.Extensions | NuGet [1.0.0-preview.6](https://www.nuget.org/packages/Azure.Core.Extensions/1.0.0-preview.6) |  |  |
+| Azure.DCAP.Windows | NuGet [1.2.0](https://www.nuget.org/packages/Azure.DCAP.Windows/1.2.0) |  |  |
+| Azure.DigitalTwins.Core | NuGet [1.0.0-preview.3](https://www.nuget.org/packages/Azure.DigitalTwins.Core/1.0.0-preview.3) | [docs](https://docs.microsoft.com/dotnet/api/overview/azure/DigitalTwins.Core-readme/) | github [1.0.0-preview.3](https://github.com/Azure/azure-sdk-for-net/tree/Azure.DigitalTwins.Core_1.0.0-preview.3/sdk/digitaltwins/Azure.DigitalTwins.Core/) |
+| Azure.Extensions.Configuration.Secrets | NuGet [1.0.0-preview.1](https://www.nuget.org/packages/Azure.Extensions.Configuration.Secrets/1.0.0-preview.1) |  |  |
+| Azure.Storage.Blobs.Batching | NuGet [12.0.0-preview.4](https://www.nuget.org/packages/Azure.Storage.Blobs.Batching/12.0.0-preview.4) |  |  |
+| Azure.Storage.Blobs.ChangeFeed | NuGet [12.0.0-preview.4](https://www.nuget.org/packages/Azure.Storage.Blobs.ChangeFeed/12.0.0-preview.4) | [docs](https://docs.microsoft.com/dotnet/api/overview/azure/Storage.Blobs.ChangeFeed-readme/) | github [12.0.0-preview.4](https://github.com/Azure/azure-sdk-for-net/tree/Azure.Storage.Blobs.ChangeFeed_12.0.0-preview.4/sdk/storage/Azure.Storage.Blobs.ChangeFeed/) |
+| Azure.Storage.Files | NuGet [12.0.0-preview.4](https://www.nuget.org/packages/Azure.Storage.Files/12.0.0-preview.4) |  |  |
+| Azure.Template | NuGet [1.0.2-preview.17](https://www.nuget.org/packages/Azure.Template/1.0.2-preview.17) |  |  |
+| dotnet-Perks.CodeGen | NuGet [2.0.1](https://www.nuget.org/packages/dotnet-Perks.CodeGen/2.0.1) |  |  |
+| Hyak.Common | NuGet [1.2.2](https://www.nuget.org/packages/Hyak.Common/1.2.2) |  |  |
+| Hyak.Common.Tracing.Etw | NuGet [1.0.2](https://www.nuget.org/packages/Hyak.Common.Tracing.Etw/1.0.2) |  |  |
+| Hyak.Common.Tracing.Log4Net | NuGet [1.0.2](https://www.nuget.org/packages/Hyak.Common.Tracing.Log4Net/1.0.2) |  |  |
+| Microsoft.AutoRest.Common | NuGet [2.4.48](https://www.nuget.org/packages/Microsoft.AutoRest.Common/2.4.48) |  | github [2.4.48](https://github.com/Azure/azure-sdk-for-net/tree/Microsoft.AutoRest.Common_2.4.48/sdk/https://github.com/Azure/autorest.common/Microsoft.AutoRest.Common/) |
+| Microsoft.Azure.ApplicationInsights | NuGet [0.9.0-preview](https://www.nuget.org/packages/Microsoft.Azure.ApplicationInsights/0.9.0-preview) |  |  |
+| Microsoft.Azure.ApplicationInsights.Query | NuGet [0.9.0-preview](https://www.nuget.org/packages/Microsoft.Azure.ApplicationInsights.Query/0.9.0-preview) |  | github [0.9.0-preview](https://github.com/Azure/azure-sdk-for-net/tree/Microsoft.Azure.ApplicationInsights.Query_0.9.0-preview/sdk/https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/applicationinsights/Microsoft.Azure.ApplicationInsights.Query/Microsoft.Azure.ApplicationInsights.Query/) |
```

Comment:
This azure-sdk bot seems to have a bug. This URL for github 0.9.0-preview is bad (contains two concatenated https:// strings). I've fixed them in https://github.com/dotnet/docs/pull/20637.